free/latest_v10_release.json

```diff
@@ -0,0 +1,27 @@
+[
+    {
+        "location": "https://download.katalon.com/free/10.0.0/release/Katalon_Studio_Windows_64-10.0.0.zip",
+        "file": "win_64",
+        "type": "ks"
+    },
+    {
+        "location": "https://download.katalon.com/free/10.0.0/release/Katalon%20Studio.dmg",
+        "file": "mac_64",
+        "type": "ks"
+    },
+    {
+        "location": "https://download.katalon.com/free/10.0.0/release/Katalon%20Studio%20Arm64.dmg",
+        "file": "mac_aarch64",
```

Comment:
Please change this to `mac_arm64` for consistency with version 9. Same for other files.